server/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -453,7 +458,7 @@ public final String getRaw(final Settings settings) {
      * @return the raw string representation of the setting value
      */
     String innerGetRaw(final Settings settings) {
-        return settings.get(getKey(), defaultValue.apply(settings));
+        return settings.get(getKey(), defaultValue.apply(settings), isListSetting());
```

Comment:
Instead of having an isListSetting, since this is already package private anyways, can this just check `instanceof ListSetting`?

Reply:
Hmm, yea it could :) -> will change to that :)

Reply:
@original-brownbear @rjernst See my [diff](
https://github.com/elastic/elasticsearch/pull/33503#issuecomment-419974678), we need to expose `isListSetting` in `AbstractScopedSettings`.

Reply:
It's implementation though can be `final` and be an instance of check against `ListSetting`.